### PR TITLE
feat(StatusListView): Add tuned ListView component and fix ScrollView srollbars

### PR DESCRIPTION
**Pull request by MishkaRogachev**
For https://github.com/status-im/status-desktop/pull/6456

### Checklist

- [x] follow [conventional commits](https://www.conventionalcommits.org/en/v1.0.0/)
  - the scope should be the component's name e.g: `feat(StatusListItem): ... `
  - when adding new components, the scope is the module e.g: `feat(StatusQ.Controls): ...`
- [x] add documentation if necessary (new component, new feature)
- [ ] update sandbox app
  - in case of new component, add new component page
  - in case of new features, add variation to existing component page
  - nice to have: add it to the demo application as well
- [ ] test changes in both light and dark theme?
- [ ] is this a breaking change?
    - [ ] use the dedicated `BREAKING CHANGE` commit message section
    - [ ] resolve breaking changes in [status-desktop](https://github.com/status-im/status-desktop)
        - [ ] (pre-merge) adapt code to breaking changes
        - [ ] (post-merge) update StatusQ submodule pointer
- [x] test changes in [status-desktop](https://github.com/status-im/status-desktop)

### CODE REVIEW

File: src/StatusQ/Core/StatusScrollView.qml [hunk 1]
```diff
@@ -20,29 +20,37 @@ import StatusQ.Controls 0.1
...
+Flickable {
+    id: flickable
+
+    property int padding: 8
+    property int topPadding: padding
+    property int bottomPadding: padding
+    property int leftPadding: padding
+    property int rightPadding: padding
+
+    property Item background: Item {}
```
Comment (caybro) [inline src/StatusQ/Core/StatusScrollView.qml:39]: `: null` is empty

**Review by caybro — CHANGES_REQUESTED** [review]

**Review by noeliaSD — APPROVED** [review]

**Review by caybro — APPROVED** [review]
Looks good!

**Review by noeliaSD — APPROVED** [review]

**Comment by MishkaRogachev** [comment]
@caybro @noeliaSD i'm thinking about renaming StatusScrollView to StatusFlickable for few reasons:
- Since Qt 5.15 Flickable have [own](https://doc-snapshots.qt.io/qt5-5.15/qml-qtquick-flickable.html#leftMargin-prop) margins
- In status desktop we often use ScrollView for decoration ListView with scrollbars. But within my changes it has no sense, because StatusListView already have scrollbars.

**Comment by caybro** [comment]
> @caybro @noeliaSD i'm thinking about renaming StatusScrollView to StatusFlickable for few reasons:
> 
> * Since Qt 5.15 Flickable have [own](https://doc-snapshots.qt.io/qt5-5.15/qml-qtquick-flickable.html#leftMargin-prop) margins
> * In status desktop we often use ScrollView for decoration ListView with scrollbars. But within my changes it has no sense, because StatusListView already have scrollbars.

I wouldn't mind either way... but notice the `margins` have been there for some time already afaik (not new in 5.15) and also the correct way to decorate a `ListView` with scrollbars would be with the `ScrollBar.vertical: ScrollBar{}` attached property

**Pull request merged**